functions/exercise17/scriptSpec.js

```diff
@@ -0,0 +1,48 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals getQuantityOfLetters*/
+
+describe("Getting the quantity of occurrences of each letter in a phrase", function () {
+    it("phrase 'a' gives one letter", function () {
+        var quantityOfLetters = getQuantityOfLetters("a");
+        expect(quantityOfLetters).toEqual([
+            { letter: "a", quantity: 1 },
+        ]);
+    });
+    it("phrase 'Aa' gives two letter 'a'", function () {
```

Comment:
Please mention in the description that the purpose of this test is to show that verification is case insensitive